docs/openapi.yaml

```diff
@@ -1,8 +1,8 @@
 openapi: 3.0.2
 servers:
-  - url: https://stacks-node-api.mainnet.stacks.co/
+  - url: https://api.mainnet.hiro.so/
     description: Mainnet
-  - url: https://stacks-node-api.testnet.stacks.co/
+  - url: https://api.testnet.hiro.so/
     description: Testnet
   - url: http://localhost:3999/
     description: Local
```

Comment:
@LakshmiLavanyaKasturi @CharlieC3 what about the `docs.stacks.co` references in this file and in various markdown files under `content/`? Should these be replaced with `docs.hiro.so`?